docroot/modules/custom/foia_api/src/Plugin/rest/resource/WebformSubmissionResource.php

```diff
@@ -0,0 +1,534 @@
+<?php
```

Comment:
I'm seeing a php Warning in the logs when submitting, both on successful and non-successful submissions:

```
Warning: end() expects parameter 1 to be array, null given in Drupal\file\Element\ManagedFile::validateManagedFile() (line 404 of /var/www/dojfoia/docroot/core/modules/file/src/Element/ManagedFile.php)  #0 /var/www/dojfoia/docroot/core/includes/bootstrap.inc(566):  _drupal_error_handler_real(2, 'end() expects p...',  '/var/www/dojfoi...', 404, Array) #1 [internal function]: _drupal_error_handler(2, 'end() expects p...',  '/var/www/dojfoi...', 404, Array) #2  /var/www/dojfoia/docroot/core/modules/file/src/Element/ManagedFile.php(404):  end(NULL) #3 [internal function]:  Drupal\file\Element\ManagedFile::validateManagedFile(Array,  Object(Drupal\Core\Form\FormState), Array) #4  /var/www/dojfoia/docroot/core/lib/Drupal/Core/Form/FormValidator.php(277):  call_user_func_array(Array, Array) #5  /var/www/dojfoia/docroot/core/lib/Drupal/Core/Form/FormValidator.php(235):  Drupal\Core\Form\FormValidator->doValidateForm(Array,  Object(Drupal\Core\Form\FormState)) #6  /var/www/dojfoia/docroot/core/lib/Drupal/Core/Form/FormValidator.php(235):  Drupal\Core\Form\FormValidator->doValidateForm(Array,  Object(Drupal\Core\Form\FormState)) #7  /var/www/dojfoia/docroot/core/lib/Drupal/Core/Form/FormValidator.php(119):  Drupal\Core\Form\FormValidator->doValidateForm(Array,  Object(Drupal\Core\Form\FormState), 'webform_submiss...') #8  /var/www/dojfoia/docroot/core/lib/Drupal/Core/Form/FormBuilder.php(571):  Drupal\Core\Form\FormValidator->validateForm('webform_submiss...',  Array, Object(Drupal\Core\Form\FormState)) #9  /var/www/dojfoia/docroot/core/lib/Drupal/Core/Form/FormBuilder.php(481):  Drupal\Core\Form\FormBuilder->processForm('webform_submiss...',  Array, Object(Drupal\Core\Form\FormState)) #10  /var/www/dojfoia/docroot/modules/contrib/webform/src/WebformSubmissionForm.php(1755):   Drupal\Core\Form\FormBuilder->submitForm(Object(Drupal\webform\WebformSubmissionForm),  Object(Drupal\Core\Form\FormState), 'api_submission') #11  /var/www/dojfoia/docroot/modules/custom/foia_api/src/Plugin/rest/resource/WebformSubmissionResource.php(183):  Drupal\webform\WebformSubmissionForm::submitValues(Array) #12 [internal function]:  Drupal\foia_api\Plugin\rest\resource\WebformSubmissionResource->post(Array,  Object(Symfony\Component\HttpFoundation\Request)) #13  /var/www/dojfoia/docroot/core/modules/rest/src/RequestHandler.php(119):  call_user_func_array(Array, Array) #14 [internal function]:  Drupal\rest\RequestHandler->handle(Object(Drupal\Core\Routing\RouteMatch),  Object(Symfony\Component\HttpFoundation\Request)) #15  /var/www/dojfoia/docroot/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(123):  call_user_func_array(Array, Array) #16  /var/www/dojfoia/docroot/core/lib/Drupal/Core/Render/Renderer.php(574):  Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}() #17  /var/www/dojfoia/docroot/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(124):   Drupal\Core\Render\Renderer->executeInRenderContext(Object(Drupal\Core\Render\RenderContext),  Object(Closure)) #18  /var/www/dojfoia/docroot/core/lib/Drupal/Core/EventSubscriber/EarlyRenderingControllerWrapperSubscriber.php(97):   Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->wrapControllerExecutionInRenderContext(Array,  Array) #19 [internal function]:  Drupal\Core\EventSubscriber\EarlyRenderingControllerWrapperSubscriber->Drupal\Core\EventSubscriber\{closure}() #20 /var/www/dojfoia/vendor/symfony/http-kernel/HttpKernel.php(144):  call_user_func_array(Object(Closure), Array) #21 /var/www/dojfoia/vendor/symfony/http-kernel/HttpKernel.php(64):  Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request),  1) #22  /var/www/dojfoia/docroot/modules/contrib/simple_oauth/src/HttpMiddleware/BasicAuthSwap.php(67):   Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request),  1, true) #23  /var/www/dojfoia/docroot/core/lib/Drupal/Core/StackMiddleware/Session.php(57):   Drupal\simple_oauth\HttpMiddleware\BasicAuthSwap->handle(Object(Symfony\Component\HttpFoundation\Request),  1, true) #24  /var/www/dojfoia/docroot/core/lib/Drupal/Core/StackMiddleware/KernelPreHandle.php(47):   Drupal\Core\StackMiddleware\Session->handle(Object(Symfony\Component\HttpFoundation\Request),  1, true) #25  /var/www/dojfoia/docroot/core/modules/page_cache/src/StackMiddleware/PageCache.php(99):   Drupal\Core\StackMiddleware\KernelPreHandle->handle(Object(Symfony\Component\HttpFoundation\Request),  1, true) #26  /var/www/dojfoia/docroot/core/modules/page_cache/src/StackMiddleware/PageCache.php(78):   Drupal\page_cache\StackMiddleware\PageCache->pass(Object(Symfony\Component\HttpFoundation\Request),  1, true) #27  /var/www/dojfoia/docroot/modules/contrib/jsonapi/src/StackMiddleware/FormatSetter.php(38):   Drupal\page_cache\StackMiddleware\PageCache->handle(Object(Symfony\Component\HttpFoundation\Request),  1, true) #28  /var/www/dojfoia/docroot/core/lib/Drupal/Core/StackMiddleware/ReverseProxyMiddleware.php(47):   Drupal\jsonapi\StackMiddleware\FormatSetter->handle(Object(Symfony\Component\HttpFoundation\Request),  1, true) #29  /var/www/dojfoia/docroot/core/lib/Drupal/Core/StackMiddleware/NegotiationMiddleware.php(50):   Drupal\Core\StackMiddleware\ReverseProxyMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request),  1, true) #30  /var/www/dojfoia/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23):   Drupal\Core\StackMiddleware\NegotiationMiddleware->handle(Object(Symfony\Component\HttpFoundation\Request),  1, true) #31 /var/www/dojfoia/docroot/core/lib/Drupal/Core/DrupalKernel.php(656):   Stack\StackedHttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request),  1, true) #32 /var/www/dojfoia/docroot/index.php(19):  Drupal\Core\DrupalKernel->handle(Object(Symfony\Component\HttpFoundation\Request)) #33 {main}.
```

Reply:
Looking into this